framework/logging.cpp

```diff
@@ -1684,11 +1777,29 @@ void KeyValuePairLogger::prepare_line_prefix()
     timestamp_prefix += test->id;
 }
 
-void KeyValuePairLogger::print(int tc)
+void KeyValuePairLogger::print(int tc, ChildExitStatus status)
 {
+    std::string skip_message = "skip";
+    std::string skip_message_from_main_thread;
+    get_skip_message(-1, skip_message_from_main_thread);
+
     logging_printf(LOG_LEVEL_QUIET, "%s_result = %s\n", test->id,
-                   state == TestSkipped ? "skip" :
+                   state == TestSkipped ? skip_message.c_str() :
                    state == TestFailed ? "fail" : "pass");
+    
+    if(status.result == TestPassed && state == TestSkipped){ // if test passed in init and skipped on all threads in run
+        logging_printf(LOG_LEVEL_QUIET, "%s_skip_category = %s\n", test->id, "RUNTIME_SKIP");
+        logging_printf(LOG_LEVEL_QUIET, "%s_skip_reason = %s\n", test->id, "Skipped in Run. Check individual thread messages");
```

Comment:
I really dislike "Skipped in Run" message. What about "All CPUs skipped during execution, check log for details."

Reply:
makes sense. I also want to replace the word "execution" with "run" or "run phase" because execution can also mean the init or run or cleanup functions.

Reply:
how about "All CPUs skipped while executing 'test_run()' function, check log for details"?

We refer to `test_run()` function in the test writing guide.

Reply:
Done 

```
result: skip
skip-category: RUNTIME_SKIP
skip-reason: All CPUs skipped while executing 'test_run()' function, check log for details
time-at-end:   { elapsed:     10.000, now: !!timestamp '2023-02-16T22:55:42Z' }
```